subcommands/version/update_self_windows.go

```diff
@@ -27,7 +27,7 @@ func replaceSelf(curExe, newExe string) error {
 		return err
 	}
 	if err := os.Rename(newExe, curExe); err != nil {
-		msg := "unable to update to the new fioctl binary. "
+		msg := "unable to update to the new Fioctl binary. "
```

Comment:
@doanac I'm not sure if we have a sentence start here, so asking you.
If we do, then I guess we need a capital letter here, and on line 46 too.

Reply:
you are correct. it's the start of a sentence

Reply:
@kprosise need to update this one too.